### DISCUSSION TITLE
Is there an Out-Gridview like command for ImportExcel?

### DISCUSSION THREAD

**Discussion by JustinGrote** [dfinke/ImportExcel#1479] [Q&A]
I would like something like Out-Gridview (I would call it `Out-ExcelView`) that will take some existing PS objects I have, export them to a temporary excel file as a table, and then open that excel file so I can manipulate it.

Does this exist today or is it a good PR candidate? I couldn't find anything specific to this use case.

**Comment by uSlackr** [comment]
Export-excel has an option `-Show` to open the file when done.

**Comment by dfinke** [comment]
You can do `Get-Service | Export-Excel` 

Creates a temp file and sets some params. Zebra striping, table coordinates, filtering etc.

![image](https://github.com/dfinke/ImportExcel/assets/67258/50d6412c-5e3f-4732-8c66-a48c31304c85)
